decidim/decidim
# Refactor CI pipelines

#### :tophat: What? Why?
This PR improves the name of several github Workflows. 
It also aims to reduce the amount of time spent by generators specs. 
It also splits the linting Pipeline into independent flows so that we could obtain a faster input when we have multiple linting offenses. 

#### Testing

1. Make sure the pipelines are green 
2. Check the time spent on generators
3. Check the linter pipelines
4. Check the names of Proposals, Meetings, Dev, Core pipelines 

### :camera: Screenshots
Brakeman:
Before:
![image](https://github.com/decidim/decidim/assets/105683/0055be01-f9b4-438f-a096-f211446a1410)

After:
![image](https://github.com/decidim/decidim/assets/105683/58838781-9f59-4ab6-9e83-b2bd22d8171b)

Generators:
Before 
![image](https://github.com/decidim/decidim/assets/105683/8b3fbb0b-c3fd-4bfd-9b40-108fc8237efb)

After:
![image](https://github.com/decidim/decidim/assets/105683/a16e440c-392f-41a1-9a8b-b4b37688859c)

:hearts: Thank you!
